scripts/create-component/plop-templates/src/components/{{componentName}}/use{{componentName}}Styles.ts.hbs

```diff
@@ -3,7 +3,6 @@ import type { {{componentName}}Slots, {{componentName}}State } from './{{compone
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 
-export const {{componentNames.propertyName}}ClassName = 'fui-{{componentName}}'
```

Comment:
You should update the conformance tests so that this does not fail for folks creating new packages https://github.com/microsoft/fluentui/blob/7878a9bace7433f6e7a4e5ee3d434ec9663b3f86/packages/react-conformance/src/defaultTests.tsx#L294

Reply:
@tomi-msft just picked up the issue to remove the tests 😁

Reply:
I'll wait for @tomi-msft's PR to be merged before merging this one:
* #24063

Reply:
@ling1726 The PR to remove the test is now merged